src/OneDrive.Sdk.Authentication.Common/MsaAuthenticationProvider.cs

```diff
@@ -20,10 +20,10 @@ namespace Microsoft.OneDrive.Sdk.Authentication
     /// </summary>
     public class MsaAuthenticationProvider : IAuthenticationProvider
     {
-        private readonly string clientId;
-        private readonly string clientSecret;
-        private readonly string returnUrl;
-        private readonly string[] scopes;
+        internal readonly string clientId;
+        internal string clientSecret;
+        internal string returnUrl;
+        internal string[] scopes;
```

Comment:
It looks like scopes is the only data type that needs to be punched through, as clientid, clientsecret, and returnurl are not used in the OnlineIdAuthenicationProvider